client/main/variableeditor.js

```diff
@@ -221,15 +221,16 @@ const EditorWidget = Backbone.View.extend({
         this.$el.empty();
         this.$el.addClass('silky-variable-editor-widget');
 
-        this.$title = $('<input class="silky-variable-editor-widget-title" type="text">').appendTo(this.$el);
+        this.$title = $('<input class="silky-variable-editor-widget-title" type="text" maxlength="50">').appendTo(this.$el);
```

Comment:
i haven't done this yet. I didn't know how to shorten the const *char to 50 and have a null character at the end.